src/garage/torch/policies/continuous_mlp_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
+This modules creates a continuous MLP policy network.
+
+A continuous MLP network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+import akro
+import torch
+
+from garage.misc.overrides import overrides
+from garage.tf.policies.base import Policy
+
+
+class ContinuousMLPPolicy(Policy):
+    """
+    This class implements a policy network.
+
+    The policy network selects action based on the state of the environment.
+    It uses neural nets to fit the function of pi(s).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 name='ContinuousMLPPolicy',
```

Comment:
I'm not convinced that we actually need a name parameter. In the tensorflow branch, it's needed to interact with the graph, but pytorch doesn't need it. Let's remove it for now, and we'll add it back if we need it in the future.

Reply:
i agree -- just remove it. does torch even accept name params?

Reply:
No.

Reply:
easy fix, then